.github/workflows/build_test.yml

```diff
@@ -26,9 +26,7 @@ jobs:
               id: vim
               with:
                   neovim: true
-                  # Use the oldest supported Neovim version here so we don't accidentally
-                  # break the extension on older versions
-                  version: v0.8.0
+                  version: v0.9.0
```

Comment:
somewhat off topic: does vscode-neovim actively check the nvim version and show a user message if it doesn't meet the requirement?

Reply:
Yes, it currently checks for v0.8. I bumped the CI version because some tests failed on my machine (running 0.9) due to some minor changes in nvim (specifically :help), and I wanted to update the test to futureproof.